html/Mogo/js/popup/popup.js

```diff
@@ -0,0 +1,39 @@
+import {components} from './components.js';
+export function addPopup() {
```

Comment:
it something more like gallery or something like this, better to name functions like this init/initialise as they init some functionality